internal/app/cf-terraforming/cmd/root.go

```diff
@@ -103,11 +103,18 @@ func init() {
 
 	rootCmd.PersistentFlags().StringVar(&resourceType, "resource-type", "", "Which resource you wish to generate")
 
-	rootCmd.PersistentFlags().StringVar(&terraformInstallPath, "terraform-install-path", ".", "Path to the Terraform installation")
+	rootCmd.PersistentFlags().StringVar(&terraformInstallPath, "terraform-install-path", ".", "Path where Terraform will be installed (if you don't have it)")
```

Comment:
this isn't quite what this flag means. the `terraform-install-path` is for referencing a working terraform installation that isn't in the immediate directory.

Reply:
it uses the existing configuration (provider config, etc) as opposed to requiring you make a new one.

Reply:
Hmmm, so maybe it should say something like: `Path to an existing, initialized terraform folder. Default is "."`?